### ISSUE TITLE
Edit discard changes dialog

### ISSUE THREAD

**Issue by eliserichards** [mozilla-lockwise/lockwise-android#818]
Depends on #628 

## Requirements 
* Should have to confirm when cancelling changes and leaving the edit view

## Visual Design
**Zeplin**
Confirm cancel - https://app.zeplin.io/project/5b6895bfe4af825140aa8dbc/screen/5c548fc431131d309f011683

**InVision**
Confirm cancel - https://mozilla.invisionapp.com/share/3SN35UHWJDX#/screens/344499814

**Comment by eliserichards** [comment]
Duplicate of #904
